WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -808,21 +812,15 @@ private void SetFeeTargetLimits()
 			}
 		}
 
-		private bool TryParseUserFee(out decimal userFee)
-		{
-			userFee = default;
-			var userFeeText = UserFeeText;
-			return
-				userFeeText is { }
-				&& !userFeeText.Contains(",")
-				&& decimal.TryParse(userFeeText, out userFee)
-				&& (userFee * 1_000) < Constants.MaxSatoshisSupply
-				&& userFee > 0;
-		}
+		public static bool TryParseUserFee(string feeText, out decimal userFee)
+			=> decimal.TryParse(feeText, NumberStyles.AllowDecimalPoint, new CultureInfo("en-US"), out userFee)
+			&& userFee >= MinRelayTxFeeRate
+			&& userFee < AbsurdlyHighFeeRate
+			&& new FeeRate(userFee) is var _;
```

Comment:
Wut? :) is bool?

Reply:
@nopara73 what is happening here? is var? it will take the return type of the property so is bool?